### DISCUSSION TITLE
Right Click with an Option to Create a File?

### DISCUSSION THREAD

**Discussion by HRNPH** [spacedriveapp/spacedrive#1851] [Feature Request]
# SpaceDrive was GREAT!
It's easy to organize my file, it's beautiful and that's why I would like to use it on a daily basis.
But one major problem was I could create a folder, But File? and maybe thing like opening a terminal in that path.
It's rather confusing for me that such features(to create file) was missing. I'm sorry in advance if this was a technical issue for it.
or it is already available, but from my pov there's no way to create empty file easily.
## Windows Create File
![image](https://github.com/spacedriveapp/spacedrive/assets/51855316/b56801b6-5d9c-4bda-b157-cad4d18802c5)
## Spacedrive
I think something like empty file creation and instantly renaming should be best (like dolphin) 
![image](https://github.com/spacedriveapp/spacedrive/assets/51855316/12d0aead-886f-4a18-9bf6-a370bf1548a3)

**Comment by utkubakir** [comment]
This is normal on macOS but as an old Windows user, I can see why this is a big missing feature. We will work on it!
